main.go

```diff
@@ -129,13 +124,13 @@ func doSlaveReport(masterAddr *net.UDPAddr) bool {
 	// it doesn't really matter what port we listen for
 	lAddr, err := net.ResolveUDPAddr("udp", ":0")
 	if err != nil {
-		llog.Fatal("error resolving report listen UDP addr", kv.Set("err", err))
+		llog.Error("error resolving report listen UDP addr", kv.Set("err", err))
 		return false
 	}
 
 	c, err := net.ListenUDP("udp", lAddr)
 	if err != nil {
-		llog.Error("error connecting to master", kv.Set("err", err))
```

Comment:
This error message was just wrong. It should be `cannot listen on local port`. This should also be an error or even maybe a fatal. Not sure if this is even possible.